lib/components/molecules/Form/FieldFileInput/FieldFileInput.js

```diff
@@ -0,0 +1,84 @@
+// @flow
```

Comment:
Can you attach screenshots in your PR ?

Reply:
![image](https://user-images.githubusercontent.com/49902237/70045732-94ba7680-15ea-11ea-8962-12dfd3927e5a.png)